tools/rxjs/rxjs_shims.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
```

Comment:
Also just realized this. This license is not in sync with what we usually have in Material. I know that this file is from `angular/angular` and they don't use `Google LLC.`. thought just checking if we want to update it.

Reply:
Fixed